bundles/org.openhab.ui/web/src/components/app.vue

```diff
@@ -406,14 +407,20 @@ export default {
         this.$f7.preloader.hide()
         this.$f7.dialog.alert('Error while signing out: ' + err)
       })
+    },
+    updateThemeOptions () {
```

Comment:
Should themeOptions be a computed property vs me sticking them in a method?  Seem like i took the cheap way out here.  Since we rely on this.$f7 , this can not be called from the "created" function anymore